README.template.md

```diff
@@ -2,33 +2,23 @@
 
 ## Packages
 
-The Kotlin SDK is available on the Sonatype Central snapshots repo:
+The Kotlin SDK is available on Maven Central:
 
 ### Gradle
 
 ```kotlin
-repositories {
-    maven("https://s01.oss.sonatype.org/content/repositories/snapshots/")
-}
-
 dependencies {
-    implementation("software.momento.kotlin:sdk:0.1.0-SNAPSHOT")
+    implementation("software.momento.kotlin:sdk:0.1.3")
```

Comment:
at some point we might want to see if we can get release please to update these but definitely not necessary for this PR

Reply:
I agree.